mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/cfg/StandalonePojoMapperSettings.java

```diff
@@ -63,6 +80,9 @@ private Radicals() {
 		public static final String SCHEMA_MANAGEMENT_STRATEGY = "schema_management.strategy";
 		public static final String MAPPING_CONFIGURER = "mapping.configurer";
 		public static final String MULTI_TENANCY_ENABLED = "mapping.multi_tenancy.enabled";
+		public static final String INDEXING_PLAN_SYNCHRONIZATION_PREFIX = "standalone.indexing.plan.synchronization.";
```

Comment:
Same as last time we discussed it: please don't add prefixes such as `standalone.` to configuration properties. We don't do that anywhere else :)

Reply:
yeah, I remember it 😃 but ... 
This as well as the  `orm-` or `pojo-` prefixes in the predefined bean names and in the new property for the ORM mapper was done because I've run into a problem that there are multiple beans with the same name defined. I had the strategies for both ORM and Standalone mappers added using the interface from the pojo-base.  
In the end, both mappers have their specific strategies, so that shouldn't be a problem anymore (I hope 🙈), but I've left the prefixes for the time being until we have a "final approach decided" 😃